x-pack/plugins/session_view/public/components/detail_panel_accordion/index.tsx

```diff
@@ -19,6 +19,8 @@ interface DetailPanelAccordionDeps {
   tooltipContent?: string;
   extraActionTitle?: string;
   onExtraActionClick?: () => void;
+  children?: ReactNode;
+  isOpen?: boolean;
```

Comment:
`isInitialOpen` would be a better name with how this prop is being used. `isOpen` is referring to giving complete control of the parent to toggle the accordion to open/close.